doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -813,8 +813,12 @@ are applied using the :tacn:`rewrite_strat` tactic.
 .. insertprodn rewstrategy rewstrategy0
 
 .. prodn::
-   rewstrategy ::= {+; @rewstrategy1 }
+   rewstrategy ::= fix @ident := @rewstrategy1
+   | {+; @rewstrategy1 }
+   rewstrategy2 ::= @rewstrategy2 ; @rewstrategy1
+   | @rewstrategy1
```

Comment:
@jfehrle How do I make `rewstrategy2` not appear at all?  I have
```
rewstrategy: [
| REPLACE rewstrategy2
| WITH LIST1 rewstrategy1 SEP ";"
]
```
in `common.edit_mlg`, and I want to have the documentation inline `rewstrategy2` as `LIST1 rewstrategy1 SEP ";"` in all uses?

Reply:
I only saw this now :-(.

For clarity, I would drop what you have above, then use

```
rewstrategy2: [
| REPLACE rewstrategy2 ";" rewstrategy1
| WITH LIST1 rewstrategy1 SEP ";"
| DELETE rewstrategy1
]

SPLICE: [
| rewstrategy2
]
```

Or you could just add this to what you have:

`rewstrategy2: [ | DELETENT ]`

but that's less robust--if another production is ever added to `rewstrategy2`, it would disappear in an invisible way whereas it would appear and be noticeable with the first approach.

I didn't test these.  Doc for these operations is in `doc/tools/docgram/README.md`.

Reply:
Thank you!  And no worries about the delay.  I guess it didn't show up because I had the PR in draft until just recently and so you were tagged but not review requested?

I don't suppose the CI is set up so that I can get the changes from running the doc update target from the web interface, without access to my laptop?

Reply:
AFAIK I should get an email anytime I'm tagged in a PR.

I have no idea about the web interface or the CI setup.

Reply:
The notice I got was because I'm a dog reviewer.

Reply:
Seems to work, thanks!